stablehlo/reference/Ops.cpp

```diff
@@ -91,6 +91,14 @@ Tensor evalFloorOp(const Tensor &operand, Type resultType) {
   return result;
 }
 
+SmallVector<Tensor> evalIfOp(const Tensor &pred, Region &trueBranch,
+                             Region &falseBranch, Scope &scope) {
+  SmallVector<Tensor> runtimeResults;
```

Comment:
No need to use this vector for empty args. Pass empty args as `{}` to the evals instead.